pkg/kubelet/network/plugins.go

```diff
@@ -166,6 +175,10 @@ func (plugin *noopNetworkPlugin) Name() string {
 	return DefaultPluginName
 }
 
+func (plugin *noopNetworkPlugin) Capabilities() uint {
+	return 0
```

Comment:
The noop plugin itself doesn't support shaping, but the kubelet core does completely outside of the plugins.  The capability is there to indicate that the plugin itself handles shaping and the core doesn't need to.